### PR TITLE
Documentation for GPU Operator 23.9.2

### PR DESCRIPTION
**Pull request by cdesiniotis**
Review HTML:

* RN: https://nvidia.github.io/cloud-native-docs/review/pr-24/gpu-operator/latest/release-notes.html
* Platform Support: https://nvidia.github.io/cloud-native-docs/review/pr-24/gpu-operator/latest/platform-support.html#gpu-operator-component-matrix

### CODE REVIEW

File: gpu-operator/platform-support.rst [hunk 1]
```diff
@@ -461,7 +461,7 @@ Support for GPUDirect RDMA
 
 Supported operating systems and NVIDIA GPU Drivers with GPUDirect RDMA.
 
-- Ubuntu 20.04 and 22.04 LTS with Network Operator 23.10.0
+- Ubuntu 20.04 and 22.04 LTS with Network Operator 24.1.0
```
Comment (cdesiniotis) [inline gpu-operator/platform-support.rst:464]: A few open questions:
- are we bumping the Network Operator version we support on OpenShift as well?
- are we bumping the Network Operator version we support with GDS as well?

File: gpu-operator/release-notes.rst [hunk 2]
```diff
@@ -115,14 +224,6 @@ Fixed issues
   This fix creates the new service account, a new cluster role, and a new cluster role binding.
   The update prevents failures with Argo CD.
 
-* Previously, duplicate image pull secrets were added to some daemon sets and caused an error
```
Comment (cdesiniotis) [inline gpu-operator/release-notes.rst:118]: Note, we added this to the release notes for 23.9.1, but it doesn't look like this fix actually made it into 23.9.1.

**Review by tariq1890 — APPROVED** [review]

File: gpu-operator/release-notes.rst (hunk 2 above)
Comment (mikemckiernan) [reply gpu-operator/release-notes.rst:118]: TY. Hrm, I'll have to figure out why I thought the update made it in.  My MR tracing skills mustn't be what I thought they were.

File: gpu-operator/platform-support.rst [hunk 3]
```diff
@@ -434,17 +434,16 @@ Support for KubeVirt and OpenShift Virtualization
...
-================    ===========   =============   =========    =============    ========
-Ubuntu 20.04 LTS    1.22---1.28   0.36+           0.59.1+
-Ubuntu 22.04 LTS    1.22---1.28   0.36+           0.59.1+
-Red Hat Core OS                                                4.11---4.14      4.13,
-                                                                                4.14
-================    ===========   =============   =========    =============    ========
+================    ===========   =============   =========    =============    ===========
+Ubuntu 20.04 LTS    1.22---1.29   0.36+           0.59.1+
+Ubuntu 22.04 LTS    1.22---1.29   0.36+           0.59.1+
+Red Hat Core OS                                                4.11---4.15      4.13, 4.15
```
Comment (mikemckiernan) [inline gpu-operator/platform-support.rst:445]: @cdesiniotis, vGPU is 4.13 and 4.15, not through 4.15?  I don't know any details, I'm just double checking.

**Review by mikemckiernan — APPROVED** [review]
Much appreciated Chris.  Thank you again.

File: gpu-operator/platform-support.rst (hunk 3 above)
Comment (cdesiniotis) [reply gpu-operator/platform-support.rst:445]: This is a typo. It should be 4.13 through 4.15.

Comment (cdesiniotis) [reply gpu-operator/platform-support.rst]: I fixed this now.

**Pull request merged**